tests/unit/parsers/test.manifestjson.js

```diff
@@ -338,7 +369,26 @@ describe('ManifestJSONParser', () => {
   });
 
   describe('enum', () => {
-    it('should only return one message', () => {
+    it('optional_permissions should only return one message', () => {
+      const addonLinter = new Linter({ _: ['bar'] });
+      const json = validManifestJSON({
+        optional_permissions: ['idle', 'wat'],
+      });
+      const manifestJSONParser = new ManifestJSONParser(
+        json,
+        addonLinter.collector
+      );
+      expect(manifestJSONParser.isValid).toEqual(false);
+      const { warnings } = addonLinter.collector;
+      // expect(warnings.length).toEqual(1);
```

Comment:
This was one of the tests that was failing bc the expected warning list is 1 but this case produced 3 warnings. I could use some guidance here @rpl